pkg/virt-operator/resource/generate/rbac/handler.go

```diff
@@ -81,6 +81,17 @@ func newHandlerClusterRole() *rbacv1.ClusterRole {
 					"update", "list", "watch",
 				},
 			},
+			{
+				APIGroups: []string{
+					"kubevirt.io",
+				},
+				Resources: []string{
+					"virtualmachineinstancemigrations",
+				},
+				Verbs: []string{
+					"get", "list", "watch",
+				},
+			},
```

Comment:
Why do we need these? The handler has a very few rules on purpose as it runs on every node. It should have very limited view on the cluster in order to not leak too much information. At the same time as the handler scales with #nodes it scalability is limited by heavy operations.

Reply:
I think this is not right. virt-handler doesn't need to get access to the vmim objects.